packages/tldraw/src/lib/Tldraw.tsx

```diff
@@ -147,7 +147,7 @@ export function Tldraw(props: TldrawProps) {
 
 	const _tools = useShallowArrayIdentity(tools)
 	const toolsWithDefaults = useMemo(
-		() => mergeArraysAndReplaceDefaults('id', allDefaultTools, _tools),
+		() => mergeArraysAndReplaceDefaults('id', _tools, allDefaultTools),
```

Comment:
admittedly i can see how this mistake can be made since in JS typically you do `[...defaults, ...overrides]` this function does the opposite which is maybe why it's a slight footgun